healthcheck/check.go

```diff
@@ -161,10 +161,7 @@ func (s *CheckState) Update(status, message string, statusCode int) error {
 
 // hasRun returns true if the check has been run and has state
 func (c *Check) hasRun() bool {
-	if c.state.LastChecked() == nil {
-		return false
-	}
-	return true
+	return !(c.state.LastChecked() == nil)
```

Comment:
Any chance you can change the above line to something that is less terse and whose order of code matches the comment for this function, such as:

	if c.state.LastChecked() != nil {
		return true
	}
	return false